src/updateDisplay.js

```diff
@@ -0,0 +1,40 @@
+import pressedAnotherOperator from "./pressedAnotherOperator";
+
+const updateDisplay = (result, button, lastEquation) => {
+  const lastCharIsOperator = "/*+-".includes(result[result.length - 1]);
+  const isOperator = "/*+-".includes(button);
+
+  const hasLastChar = (char) => {
+    return result.substr(-1) === char;
+  };
+
+  if (button === "-" && hasLastChar("-")) {
+    button = ""; //no double negatives allowed
+  }
+
+  if (lastEquation !== "" && button !== "=") {
+    //start new equation
+    result = "";
+  }
+
+  if (result === "") {
+    if (isOperator) {
```

Comment:
@grimxreaper why the nested `if`?

Reply:
I ended up removing this code because it became superfluous.